modules/mobfoxpbBidAdapter.js

```diff
@@ -0,0 +1,136 @@
+import { isFn, deepAccess } from '../src/utils.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
```

Comment:
Do you mind just pulling in the needed utils functions? We removed `import *` from all adapters 🙏 Thanks

Reply:
@ChrisHuie thanks, updated